Frontend/Models/ProductPrice.cs

```diff
@@ -13,7 +13,7 @@ public class ProductPrice
         public int Id { get; set; }
 
         [Required]
-        [Range(1, 100000000000, ErrorMessage = "Price must be more then -1.")]
+        [Range(1, 100000, ErrorMessage = "Price must be more then -1.")]
```

Comment:
Might be better to write `Price must be between 1 and 100000.` since you define a range between 1 and 100000 in the attribute.

Reply:
Fixed! Nice one, missed that one